{"uid":"dd8aa718413ee3d","name":"Open Dashboard Menu","fullName":"test.test_customers#test_cannot_submit_form","historyId":"a58d8ec7338bea0cc4cc9fcaf52bc271","time":{"start":1729087664243,"stop":1729087680868,"duration":16625},"description":"\n    Test that the form cannot be submitted without required fields\n    ","descriptionHtml":"<pre><code>Test that the form cannot be submitted without required fields\n</code></pre>\n","status":"broken","statusMessage":"selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: move target out of bounds\n  (Session info: chrome=83.0.4103.106)\nStacktrace:\n#0 0x559a0f9db579 <unknown>","statusTrace":"appium_driver = <appium.webdriver.webdriver.WebDriver (session=\"3cc81c8d-8641-468d-9b84-aaca112e7ff4\")>\n\n    @allure.title(\"Open Dashboard Menu\")\n    def test_cannot_submit_form(appium_driver):\n        \"\"\"\n        Test that the form cannot be submitted without required fields\n        \"\"\"\n        with allure.step(\"Open form\"):\n            try:\n                appium_driver.find_element(\n                    by=AppiumBy.XPATH,\n                    value='//android.widget.Button[@text=\"CREATE CUSTOMER\"]',\n                ).click()\n            except NoSuchElementException:\n                pass\n    \n            time.sleep(1)\n            actions = ActionChains(appium_driver)\n            # override as 'touch' pointer action\n            actions.w3c_actions = ActionBuilder(appium_driver, mouse=PointerInput(interaction.POINTER_TOUCH, \"touch\"))\n            actions.w3c_actions.pointer_action.move_to_location(870, 2600)\n            actions.w3c_actions.pointer_action.pointer_down()\n            actions.w3c_actions.pointer_action.pause(2)\n            actions.w3c_actions.pointer_action.move_to_location(870, 1000)\n            actions.w3c_actions.pointer_action.release()\n>           actions.perform()\n\ntest/test_customers.py:99: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/lib/python3.10/site-packages/selenium/webdriver/common/action_chains.py:94: in perform\n    self.w3c_actions.perform()\n../../../.local/lib/python3.10/site-packages/selenium/webdriver/common/actions/action_builder.py:96: in perform\n    self.driver.execute(Command.W3C_ACTIONS, enc)\n../../../.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <appium.webdriver.errorhandler.MobileErrorHandler object at 0x7f76a014d450>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"move target out of bounds\",\"message\":\"move target out of bounds\\\\n  (Session info: chrome=83.0.4103.106)\",\"stacktrace\":\"#0 0x559a0f9db579 <unknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"\n        https://www.w3.org/TR/webdriver/#errors\n        \"\"\"\n        payload = response.get('value', '')\n        if isinstance(payload, dict):\n            payload_dict = payload\n        else:\n            try:\n                payload_dict = json.loads(payload)\n            except (json.JSONDecodeError, TypeError):\n                return\n            if not isinstance(payload_dict, dict):\n                return\n        value = payload_dict.get('value')\n        if not isinstance(value, dict):\n            return\n        error = value.get('error')\n        if not error:\n            return\n    \n        message = value.get('message', error)\n        stacktrace = value.get('stacktrace', '')\n        # In theory, we should also be checking HTTP status codes.\n        # Java client, for example, prints a warning if the actual `error`\n        # value does not match to the response's HTTP status code.\n        exception_class: Type[sel_exceptions.WebDriverException] = ERROR_TO_EXC_MAPPING.get(\n            error, sel_exceptions.WebDriverException\n        )\n        if exception_class is sel_exceptions.WebDriverException and message:\n            if message == 'No such context found.':\n                exception_class = appium_exceptions.NoSuchContextException\n            elif message == 'That command could not be executed in the current context.':\n                exception_class = appium_exceptions.InvalidSwitchToTargetException\n    \n        if exception_class is sel_exceptions.UnexpectedAlertPresentException:\n            raise sel_exceptions.UnexpectedAlertPresentException(\n                msg=message,\n                stacktrace=format_stacktrace(stacktrace),\n                alert_text=value.get('data'),\n            )\n>       raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\nE       selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: move target out of bounds\nE         (Session info: chrome=83.0.4103.106)\nE       Stacktrace:\nE       #0 0x559a0f9db579 <unknown>\n\n../../../.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:125: MoveTargetOutOfBoundsException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"appium_driver","time":{"start":1729087606553,"stop":1729087639376,"duration":32823},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"description":"\n    Test that the form cannot be submitted without required fields\n    ","status":"broken","statusMessage":"selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: move target out of bounds\n  (Session info: chrome=83.0.4103.106)\nStacktrace:\n#0 0x559a0f9db579 <unknown>","statusTrace":"appium_driver = <appium.webdriver.webdriver.WebDriver (session=\"3cc81c8d-8641-468d-9b84-aaca112e7ff4\")>\n\n    @allure.title(\"Open Dashboard Menu\")\n    def test_cannot_submit_form(appium_driver):\n        \"\"\"\n        Test that the form cannot be submitted without required fields\n        \"\"\"\n        with allure.step(\"Open form\"):\n            try:\n                appium_driver.find_element(\n                    by=AppiumBy.XPATH,\n                    value='//android.widget.Button[@text=\"CREATE CUSTOMER\"]',\n                ).click()\n            except NoSuchElementException:\n                pass\n    \n            time.sleep(1)\n            actions = ActionChains(appium_driver)\n            # override as 'touch' pointer action\n            actions.w3c_actions = ActionBuilder(appium_driver, mouse=PointerInput(interaction.POINTER_TOUCH, \"touch\"))\n            actions.w3c_actions.pointer_action.move_to_location(870, 2600)\n            actions.w3c_actions.pointer_action.pointer_down()\n            actions.w3c_actions.pointer_action.pause(2)\n            actions.w3c_actions.pointer_action.move_to_location(870, 1000)\n            actions.w3c_actions.pointer_action.release()\n>           actions.perform()\n\ntest/test_customers.py:99: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/lib/python3.10/site-packages/selenium/webdriver/common/action_chains.py:94: in perform\n    self.w3c_actions.perform()\n../../../.local/lib/python3.10/site-packages/selenium/webdriver/common/actions/action_builder.py:96: in perform\n    self.driver.execute(Command.W3C_ACTIONS, enc)\n../../../.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <appium.webdriver.errorhandler.MobileErrorHandler object at 0x7f76a014d450>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"move target out of bounds\",\"message\":\"move target out of bounds\\\\n  (Session info: chrome=83.0.4103.106)\",\"stacktrace\":\"#0 0x559a0f9db579 <unknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"\n        https://www.w3.org/TR/webdriver/#errors\n        \"\"\"\n        payload = response.get('value', '')\n        if isinstance(payload, dict):\n            payload_dict = payload\n        else:\n            try:\n                payload_dict = json.loads(payload)\n            except (json.JSONDecodeError, TypeError):\n                return\n            if not isinstance(payload_dict, dict):\n                return\n        value = payload_dict.get('value')\n        if not isinstance(value, dict):\n            return\n        error = value.get('error')\n        if not error:\n            return\n    \n        message = value.get('message', error)\n        stacktrace = value.get('stacktrace', '')\n        # In theory, we should also be checking HTTP status codes.\n        # Java client, for example, prints a warning if the actual `error`\n        # value does not match to the response's HTTP status code.\n        exception_class: Type[sel_exceptions.WebDriverException] = ERROR_TO_EXC_MAPPING.get(\n            error, sel_exceptions.WebDriverException\n        )\n        if exception_class is sel_exceptions.WebDriverException and message:\n            if message == 'No such context found.':\n                exception_class = appium_exceptions.NoSuchContextException\n            elif message == 'That command could not be executed in the current context.':\n                exception_class = appium_exceptions.InvalidSwitchToTargetException\n    \n        if exception_class is sel_exceptions.UnexpectedAlertPresentException:\n            raise sel_exceptions.UnexpectedAlertPresentException(\n                msg=message,\n                stacktrace=format_stacktrace(stacktrace),\n                alert_text=value.get('data'),\n            )\n>       raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\nE       selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: move target out of bounds\nE         (Session info: chrome=83.0.4103.106)\nE       Stacktrace:\nE       #0 0x559a0f9db579 <unknown>\n\n../../../.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:125: MoveTargetOutOfBoundsException","steps":[{"name":"Open form","time":{"start":1729087664243,"stop":1729087680867,"duration":16624},"status":"broken","statusMessage":"selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: move target out of bounds\n  (Session info: chrome=83.0.4103.106)\nStacktrace:\n#0 0x559a0f9db579 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Appium_test/Appium_test/test/test_customers.py\", line 99, in test_cannot_submit_form\n    actions.perform()\n  File \"/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/common/action_chains.py\", line 94, in perform\n    self.w3c_actions.perform()\n  File \"/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/common/actions/action_builder.py\", line 96, in perform\n    self.driver.execute(Command.W3C_ACTIONS, enc)\n  File \"/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/home/runner/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py\", line 125, in check_response\n    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"hasContent":true},"afterStages":[{"name":"appium_driver::fin","time":{"start":1729087762820,"stop":1729087763722,"duration":902},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"feature","value":"Test Customers Table"},{"name":"epic","value":"Customers Epic"},{"name":"parentSuite","value":"test"},{"name":"suite","value":"test_customers"},{"name":"host","value":"fv-az1986-129"},{"name":"thread","value":"2930-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.test_customers"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"dd8aa718413ee3d.json","parameterValues":[]}